src/test/java/com/superdevs/HealthOMeter/HealthOMeterApplicationTests.java

```diff
@@ -21,7 +21,7 @@
 
 	@BeforeAll
 	static void loadTestData(){
-		context = new AnnotationConfigApplicationContext(CalculatorManager.class, CalculatorService.class, WhrCalculator.class);
+		context = new AnnotationConfigApplicationContext(CalculatorManager.class, CalculatorService.class, WHRCalculator.class);
 		calculatorService = context.getBean(CalculatorService.class);
 	}
 
```

Comment:
This class should only be used for loadContext 
 e.g.

```
@SpringBootTest
class HealthOMeterApplicationTests.java{

    @Test
    void contextLoads() {
    }
}
```

if you want to make test for specific classes, method etc. then create new package and classes that will check your code.

Reply:
Disagree. I mean by default you are right, but in future we will make refactor and split it.